sdk/servicebus/service-bus/test/utils/aadUtils.ts

```diff
@@ -1,111 +1,64 @@
-import msRestNodeAuth from "@azure/ms-rest-nodeauth";
-import { ServiceBusManagementClient } from "@azure/arm-servicebus";
-import { SBQueue, SBTopic, SBSubscription } from "@azure/arm-servicebus/esm/models";
-import { EnvVarKeys, getEnvVars } from "./envVarUtils";
-
-const _loginWithServicePrincipalSecret = msRestNodeAuth.loginWithServicePrincipalSecret;
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
 
-export const loginWithServicePrincipalSecret = _loginWithServicePrincipalSecret;
+import { ServiceBusAtomManagementClient } from "../../src/serviceBusAtomManagementClient";
+import { EnvVarKeys, getEnvVars } from "./envVarUtils";
+import { QueueOptions, TopicOptions, SubscriptionOptions } from "../../src";
 
-export async function recreateQueue(queueName: string, parameters: SBQueue): Promise<void> {
+export async function recreateQueue(queueName: string, parameters: QueueOptions): Promise<void> {
   const env = getEnvVars();
-  await msRestNodeAuth
-    .loginWithServicePrincipalSecret(
-      env[EnvVarKeys.AAD_CLIENT_ID],
-      env[EnvVarKeys.AAD_CLIENT_SECRET],
-      env[EnvVarKeys.AAD_TENANT_ID]
-    )
-    .then(async (creds) => {
-      const client = await new ServiceBusManagementClient(
-        creds,
-        env[EnvVarKeys.AZURE_SUBSCRIPTION_ID]
-      );
-      await client.queues.deleteMethod(
-        env[EnvVarKeys.RESOURCE_GROUP],
-        getNamespace(env[EnvVarKeys.SERVICEBUS_CONNECTION_STRING]),
-        queueName,
-        function(error: any): void {
-          if (error) throw error.message;
-        }
-      );
-      await client.queues.createOrUpdate(
-        env[EnvVarKeys.RESOURCE_GROUP],
-        getNamespace(env[EnvVarKeys.SERVICEBUS_CONNECTION_STRING]),
-        queueName,
-        parameters,
-        function(error: any): void {
-          if (error) throw error.message;
-        }
-      );
-    });
+  if (env[EnvVarKeys.CLEAN_NAMESPACE]) {
```

Comment:
In the `testUtils.ts` file from where the recreate methods are called, we are doing a check on the `CLEAN_NAMESPACE` env value. Is there a need to repeat the same check here?